igvc_gazebo/nodes/ground_truth/main.cpp

```diff
@@ -95,14 +125,7 @@ int main(int argc, char** argv)
   utm_to_odom.setOrigin(
       tf::Vector3(utm_x - g_og_pose.pose.pose.position.x, utm_y - g_og_pose.pose.pose.position.y, 0.0));
   utm_to_odom.setRotation(tf::Quaternion(0.0, 0.0, 0.0, 1.0));
-  tf::TransformBroadcaster br;
 
-  ros::Rate rate(20);
-  while (ros::ok())
-  {
-    br.sendTransform(tf::StampedTransform(utm_to_odom, ros::Time::now(), "utm", "odom"));
-    ros::spinOnce();
-    rate.sleep();
-  }
+  ros::Timer utm_timer = nh.createTimer(ros::Duration(5.0), boost::bind(utm_callback, _1, utm_to_odom.inverse()));
```

Comment:
make this duration less. Otherwise simulation takes 5 seconds to load the initial waypoints file.

Reply:
Changed to 1 second.